gamestonk_terminal/cryptocurrency/coingecko/pycoingecko_overview_controller.py

```diff
@@ -0,0 +1,267 @@
+"""Cryptocurrency Controller"""
+__docformat__ = "numpy"
+# pylint: disable=R0904, C0302
+import argparse
+import pandas as pd
+from prompt_toolkit.completion import NestedCompleter
+from gamestonk_terminal import feature_flags as gtff
+from gamestonk_terminal.helper_funcs import get_flair
+from gamestonk_terminal.menu import session
+from gamestonk_terminal.cryptocurrency.coingecko import pycoingecko_view
+
+
+class GeckoController:
+
+    CHOICES = [
+        "help",
+        "q",
+        "quit",
+        "global",
+        "coins",
+        "defi",
+        "news",
+        "trending",
+        "most_voted",
+        "most_visited",
+        "top_volume",
+        "recently",
+        "sentiment",
+        "gainers",
+        "losers",
+        "stables",
+        "yfarms",
+        "top_defi",
+        "top_dex",
+        "top_nft",
+        "nft_today",
+        "nft_market",
+        "exchanges",
+        "ex_rates",
+        "platforms",
+        "products",
+        "indexes",
+        "derivatives",
+        "categories",
+        "hold",
+        "hold_comp",
+    ]
+
+    def __init__(self):
+        """CONSTRUCTOR"""
+
+        self._gecko_parser = argparse.ArgumentParser(add_help=False, prog="gecko")
+        self._gecko_parser.add_argument("cmd", choices=self.CHOICES)
+        self.current_coin = None
+        self.current_currency = None
+        self.current_df = pd.DataFrame()
+        self.source = ""
+
+    def print_help(self):
+        """Print help"""
+        print("\nCryptocurrency:")
+        print("   help            show this menu again")
+        print("   q               quit this menu, and shows back to main menu")
+        print("   quit            quit to abandon program")
+        print("")
+        print("Coingecko Overview: ")
+        print("   global          show global crypto market info")
+        print("   coins           show coins available on CoinGecko")
+        print("   defi            show global DeFi market info")
+        print("   trending        show trending coins on CoinGecko")
+        print("   most_voted      show most voted coins on CoinGecko")
+        print("   most_visited    show most visited coins on CoinGecko")
+        print("   top_volume      show coins with highest volume on CoinGecko")
+        print("   recently        show recently added on CoinGecko")
```

Comment:
I think in this command you should get rid of the URL, or post it in a different way. 

E.g. could be another command for the user to access this url, like:
```
url Fabric
```
And you get: https://www.coingecko.com/en/coins/fabric

This is so that we avoid the columns getting messed up due to size of table